File: Module 5/51_Linear Search/C/LinearSearch.c
#include <stdio.h>

int LinearSearch(int arr[], int target) {
    for(int i=0; i<10; i++) {
        if (arr[i] == target) {
            return i;
        }
    }
    return -1;
}

void main() {
    int arr[10] = {27,45,32,98,78,67,54,24,51,52};
    int target = 54;

    int index = LinearSearch(arr, target);

    if (index>-1)
        printf("The target value has been found at index %d.", index);
    else
        printf("The target value has not been found.");
}

File: Module 5/51_Linear Search/Java/LinearSearch.java
public class LinearSearch {
    public static int linearSearch(int arr[], int target) {
        for(int i = 0; i < arr.length; i++) {
            if (arr[i] == target) {
                return i;
            }
        }
        return -1;
    }

    public static void main(String[] args) {
        int arr[] = {27, 45, 32, 98, 78, 67, 54, 24, 51, 52};
        int target = 54;

        int index = linearSearch(arr, target);

        if (index > -1)
            System.out.println("The target value has been found at index " + index + ".");
        else
            System.out.println("The target value has not been found.");
    }
}
